tests/e2escenarios/e2e_java_test.go

```diff
@@ -104,7 +109,7 @@ var _ = Describe("odo java e2e tests", func() {
 			routeURL := helper.DetermineRouteURL(context)
 
 			// Ping said URL
-			helper.HttpWaitFor(routeURL, "HTTP Booster", 90, 1)
+			helper.HttpWaitFor(routeURL, "HTTP Booster", 300, 1)
```

Comment:
Max range of 90s is enough on any platform i guess. Any reason you increased the range to 300s

Reply:
> Max range of 90s is enough on any platform i guess. Any reason you increased the range to 300s

@amitkrout  Yes, I tested in OCP/Z , It failed everytime with 90s and successed with 300s. 300s is a safe value not the minium value.

Reply:
> @amitkrout Yes, I tested in OCP/Z , It failed everytime with 90s and successed with 300s. 300s is a safe value not the minium value.

@zhengxiaomei123 If this is the case with OCP/Z I would suggest you to check 2-3 times with consecutive run, to avoid future flakes.

Reply:
@prietyc123 I will retry more times. But I had tested it many times before. I got the error not once and not one case. I debug the error and got the solution of extending timeout.

Reply:
Changes looks good to me but I would say please confirm a consecutive run with `300s`.

/lgtm